test/e2e_node/restart_test.go

```diff
@@ -118,5 +142,93 @@ var _ = framework.KubeDescribe("Restart [Serial] [Slow] [Disruptive]", func() {
 				By(fmt.Sprintf("Docker restart test passed with %d pods", len(postRestartRunningPods)))
 			})
 		})
+		Context("Live restore", func() {
+
+			It("should restore container when live restore is on", func() {
+				framework.SkipIfContainerRuntimeIs("rkt")
+
+				// Skip when docker engine version is less than 1.12
+				dockerVersion, err := getDockerVersion()
+				dockerRegex, _ := regexp.Compile(`1\.12\.[0-9]+`)
+				if !dockerRegex.Match([]byte(dockerVersion)) {
+					framework.Skipf("Live restore isn't suported before Docker Engine version 1.12")
+				}
+
+				// skip when docker live-restore is not enabled
+				dockerPid, err := ioutil.ReadFile("/var/run/docker.pid")
+				framework.ExpectNoError(err, "Failed to get docker daemon PID: %v, stdout: %q", err, string(dockerPid))
+
+				dockerdCmd, _ := ioutil.ReadFile("/proc/" + string(dockerPid) + "/cmdline")
+
+				daemonConfFile := "/etc/docker/daemon.json"
+				_, err = os.Stat(daemonConfFile)
+				var inConfFile bool
+				if !os.IsNotExist(err) {
+					opt := new(DockerDaemonOption)
+					data, err := ioutil.ReadFile(daemonConfFile)
+					err = json.Unmarshal(data, &opt)
+					framework.ExpectNoError(err, "Failed to parse docker config file %q: %v", daemonConfFile, err)
+					inConfFile = opt.Live_restore
+				}
+				asOption := strings.Contains(string(dockerdCmd), "--live-restore")
```

Comment:
hmm... i wish it was possible to know if live restore was enabled via call to `docker info`

@mrunalp - do you know if that is possible?